cli/inputhandler/options_esdt.cpp

```diff
@@ -0,0 +1,65 @@
+#include "options_esdt.h"
+
+OptionsESDT::OptionsESDT() :
+        m_optionsIssue("ESDT", "Issue ESDT"),
+        m_optionsTransfer("ESDT", "Transfer ESDT")
+{
+    initOptions();
+}
+
+cxxopts::Options OptionsESDT::issue() const
+{
+    return m_optionsIssue;
+}
+
+cxxopts::Options OptionsESDT::transfer() const
+{
+    return m_optionsTransfer;
+}
+
+std::string OptionsESDT::help() const
+{
+    return m_optionsTransfer.help() + "\n" +
+           m_optionsIssue.help();
+}
+
+void OptionsESDT::initOptions()
+{
+    initOptionsIssue();
+    initOptionsTransfer();
+}
+
+void OptionsESDT::initOptionsIssue()
+{
+    m_optionsIssue.add_options("issue") // esdt issue group
+            ("token", "Token id", cxxopts::value<std::string>())
+            ("ticker", "Token ticker", cxxopts::value<std::string>())
+            ("supply", "Initial supply", cxxopts::value<std::string>())
+            ("dec", "Number of decimals", cxxopts::value<std::string>())
+            ("gas-price", "Gas price", cxxopts::value<uint64_t>())
+            ("pem", "Sender's pem keyfile", cxxopts::value<std::string>())
+            ("nonce", "Sender account's nonce", cxxopts::value<uint64_t>())
+            ("can-freeze", "Can freeze property (default: false)", cxxopts::value<bool>()->default_value("false"))
```

Comment:
`[Can freeze] property`, `[Can wipe] property` (some kind of separators, could be quotes as well). Or simply use a common (and less detailed) description such as `ESDT property`. Or... as it is, but remove "property" :)

Reply:
> [Can freeze] property, [Can wipe] property

I applied this suggestion